src/wasm-lib/prelude.kcl

```diff
@@ -1,3 +1,9 @@
 fn hey_from_one_of_the_devs_of_the_past_i_wish_you_a_great_day_and_maybe_gave_you_a_little_smile_as_youve_found_this = () => {
   return 42
 }
+
+fn angledLine = (args, segment_group, tag) => {
```

Comment:
do we need to mark tag as optional?

Reply:
it would be with a ? I think

Reply:
https://github.com/KittyCAD/modeling-app/pull/1669/files#diff-b007215ec7d21c68d6a4487af28c6a25fa8cf5fc95ae008e07e5680261cab1f1L11

Reply:
getting the docs out of these will be harder but I think we can figure it out, the infra around this was better in the old executor even for kcl stdlib as kcl

Reply:
We have no idea yet, WIP with @adamchalmers 

I _think_ technically all of them are "optional" but the very-very low level calls will fail.

Reply:
without the question mark the ast will not know if its optional therefore any docs we do generate will not know either

Reply:
also one bug I fixed twice with the old executor was making sure for kcl stdlib written in kcl that the errors surfaced thru the executor had the correct source range, just something to keep in mind, meaning making sure the error never says the source range for the prelude since no one will have any idea wtf that is

Reply:
I'm just not aware of any mechanisms we have for optional values directly in KCL - if ? exists than that one less thing for us to do :) Yes you're right tag should be optional - sorry, I thought that wasn't the primary question

Reply:
Ah good to mention! We haven't added source ranges back in yet but will keep it in mind

Reply:
? already works :) so i was basically saying use it ahha

Reply:
its a hidden thing